tools/cockpit.spec.in

```diff
@@ -53,20 +49,17 @@ Summary:        Web Console for Linux servers
 License:        LGPLv2+
 URL:            https://cockpit-project.org/
 
-Version:        0
-%if %{defined wip}
-Release:        1.%{wip}%{?dist}
-Source0:        cockpit-%{version}.tar.xz
-%else
-Release:        1%{?dist}
-Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{version}/cockpit-%{version}.tar.xz
-%endif
+Version:        @VERSION@
+Release:        @RELEASE@
+Source0:        @PATH@cockpit-@VERSION@.tar.xz
 
 # in RHEL 8 the source package is duplicated: cockpit (building basic packages like cockpit-{bridge,system})
 # and cockpit-appstream (building optional packages like cockpit-{pcp})
 # This split does not apply to EPEL/COPR.
 # In Fedora ELN/RHEL 9+ there is just one source package, which ships rpms in both BaseOS and AppStream
-%if 0%{?rhel} == 8 && 0%{?epel} == 0
+# We also provide an override mechanism if you want to build all packages.
+%define build_all @BUILD_ALL@
+%if 0%{?rhel} == 8 && 0%{?epel} == 0 && !%{build_all}
```

Comment:
Just FYI: You can use `%{?varname}` to accept undefined macro names.

Reply:
Ya.  I managed to learn a bit about how this works today... but since we're not passing this in via `--define`, it will actually always be defined, from the line just above...